pkgs/development/python-modules/testbook/default.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  fetchFromGitHub,
+  nbformat,
+  nbclient,
+  ipykernel,
+  pandas,
+  pytest,
+  traitlets,
+}:
+buildPythonPackage rec {
+  pname = "testbook";
+  version = "0.4.2";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "nteract";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-qaDgae/5TRpjmjOf7aom7TC5HLHp0PHM/ds47AKtq8U=";
+  };
+
+  propagatedBuildInputs = [
+    nbclient
+    nbformat
+  ];
+
+  checkInputs = [
+    ipykernel
+    pandas
+    pytest
```

At first glance it looks like that `pytestCheckHook` could be used. Could you please check that?

```suggestion
    pytestCheckHook
```

---

Yup it works fine.